src/modules/keycloak-administration/entity/user-lock.entity.ts

```diff
@@ -1,10 +1,17 @@
-import { Entity, Property, ManyToOne, Rel } from '@mikro-orm/core';
+import { Entity, Property, ManyToOne, Rel, Cascade } from '@mikro-orm/core';
 import { TimestampedEntity } from '../../../persistence/timestamped.entity.js';
 import { PersonEntity } from '../../person/persistence/person.entity.js';
 
 @Entity({ tableName: 'user_lock' })
 export class UserLockEntity extends TimestampedEntity {
-    @ManyToOne({ entity: () => PersonEntity })
+    @ManyToOne({
+        entity: () => PersonEntity,
+        fieldName: 'person_id',
+        columnType: 'uuid',
+        cascade: [Cascade.REMOVE],
```

Comment:
I belive this should not be set to `Cascade.REMOVE`.
If a UserLockEntity gets removed, it should not remove the user.

Reply:
Me and @pkleybolte talked about this when implementing cascading for Personenkontext with Person and cascading only works this way smh. Deleting userLocks doesn't delete Person here anyways (Tried it many times) and removing this part of the code in the user-lock.entity.ts persists the records there even when the person is deleted.

Reply:
Weird. Reading the MikroORM docs I came to the other conclusion, but if you've tested and confirmed it thats fine by me.